de.php

```diff
@@ -130,13 +130,13 @@
     "Return" => "Zurück",
 
     // Badges
-    "Achievement top 50 for" => "Top 250 Bonussystem Badges für", // "Achievement top 50 for :hotel:"
+    "Achievement top 50 for" => "Top 50 Bonussystem Badges für", // "Achievement top 50 for :hotel:"
```

Comment:
These numbers are actually 250, but I'll change them and the comment to be more clear.